core/application-crypto/src/ed25519.rs

```diff
@@ -0,0 +1,75 @@
+// Copyright 2019 Parity Technologies (UK) Ltd.
+// This file is part of Substrate.
+
+// Substrate is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Substrate is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Substrate.  If not, see <http://www.gnu.org/licenses/>.
+
+//! Sr25519 crypto types.
+
+use crate::{RuntimePublic, KeyTypeId};
+
+pub use primitives::ed25519::*;
+
+mod app {
+	use crate::key_types::ED25519;
+	crate::app_crypto!(super, ED25519);
```

Comment:
i'l like to remove this entirely in the not too distant future. is anything using it?

Reply:
I use it in some tests. But yeah, I could probably do this in the test itself.